shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -95,6 +99,12 @@ public boolean supportReauthSignoutEnabled() {
                 .equals("true");
     }
 
+    public boolean supportAuthenticationAttemptsMigration() {
+        return System.getenv()
+                .getOrDefault("SUPPORT_AUTHENTICATION_ATTEMPTS_MIGRATION", String.valueOf(false))
```

Comment:
Can we change the name of the feature switch to be `AUTHENTICATION_ATTEMPTS_SERVICE_ENABLED` as per the naming standard we discussed in dev chat a few weeks ago.